### DISCUSSION TITLE
feat: connect Human-GEM with more measured metabolites in human cell

### DISCUSSION THREAD

**Discussion by hongzhonglu** [SysBioChalmers/Human-GEM#317] [Ideas]
As recorded in https://hmdb.ca/statistics#metabolite-statistics, there are 18,609 detected and quantified metabolites for human cell while in current Human-GEM, there are only about 8458 metabolites, thus it is essential to develop a generic pipeline to add new reactions based on the metabolomics studies so that Human-GEM can better represent the human metabolic activities.

**Comment by haowang-bioinfo** [comment]
@hongzhonglu thanks for posting this interesting issue. Look forward to a more detail plan for implementation.
